src/lib/expansion-panel/expansion-panel-core.ts

```diff
@@ -22,7 +24,16 @@ export class ExpansionPanelCore implements IExpansionPanelCore {
   public initialize(): void {
     this._adapter.addHeaderListener('click', this._clickListener);
     this._adapter.addHeaderListener('keydown', this._keydownListener);
+    this._adapter.tryLocateTargetButton(this._targetButton);
+    this._adapter.addTargetButtonListener('click', this._clickListener);
+    this._adapter.addTargetButtonListener('keydown', this._keydownListener);
     this._adapter.setAnimationCompleteListener(this._animationCompleteListener);
+    this._adapter.initializeAccessibility();
+    this._adapter.updateAriaExpanded(this._open);
+  }
+
+  public destroy(): void {
+    this._adapter.detachAria();
```

Comment:
We should attempt to remove the listeners from the detached trigger element when the expansion panel is destroyed.